### DISCUSSION TITLE
github pages stop rendering my jekyll site

### DISCUSSION THREAD

**Discussion by mzunairtariq** [community/community#34481] [Other Feature Feedback, Questions, & Ideas]
Hello Sir/Madam
I am facing a problem My site was working, but when I added the Google analytics ID, it stopped working.

https://github.com/mzunairtariq/mzunairtariq.github.io

**Comment by yoannchaudet** [comment]
In your repository settings for Pages, you have your source set to track the `master` branch. You also have a workflow you have defined in your repository.

For each push on your `master` branch you have two workflows competing for a build of your site.

Start by changing your source to "GitHub Actions" (in your repository settings). This should help and allow your [workflow](https://github.com/mzunairtariq/mzunairtariq.github.io/blob/master/.github/workflows/pages-deploy.yml) to take precedence.

**Reply by mzunairtariq** [reply]
Thanks a lot Sir, it works ✨
